modules/auxiliary/admin/kerberos/get_ticket.rb

```diff
@@ -142,12 +148,13 @@ def init_authenticator(options = {})
   def action_get_tgt
     print_status("#{peer} - Getting TGT for #{datastore['USER']}@#{datastore['DOMAIN']}")
 
+    # Never attempt to use the kerberos cache when requesting a kerberos TGT, to ensure a request is made
     authenticator = init_authenticator({ ticket_storage: kerberos_ticket_storage(read: false) })
     authenticator.request_tgt_only
   end
 
   def action_get_tgs
-    authenticator = init_authenticator(options)
+    authenticator = init_authenticator({ ticket_storage: kerberos_ticket_storage(read: true) })
```

Comment:
If kwargs are specified, both default to true, so it's only necessary to specify which options should be disabled (since both read and write default to enabled). If the intention is for the cache mode here to be read/write, it'd be more readable and clear represented as:

```suggestion
    authenticator = init_authenticator({ ticket_storage: kerberos_ticket_storage(read: true, write: true) })
```

Reply:
Updated, thanks! :+1: